### ISSUE TITLE
🐛 `useSemanticElements` should ignore inline SVG with `role="img"`

### ISSUE THREAD

**Issue by Conaclos** [biomejs/biome#3828]
### Environment information

```block
Main
```

### What happened?

Inline SVG with the role `img` is reported by the rule ([Playground](https://biomejs.dev/playground/?lintRules=all&files.main.jsx=ZQB4AHAAbwByAHQAIABjAG8AbgBzAHQAIABDACAAPQAgADwAcwB2AGcAIAByAG8AbABlAD0AIgBpAG0AZwAiACAAYQByAGkAYQAtAGwAYQBiAGUAbAA9ACIARABlAHMAYwByAGkAcAB0AGkAbwBuACAAbwBmACAAeQBvAHUAcgAgAFMAVgBHACAAaQBtAGEAZwBlACIAPgAKADwALwBzAHYAZwA%2BADsA)).
Adding `role="img"` on an inline SVG is a [recommendation from MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role#svg_and_roleimg).

### Expected result

We should ignore `role="img"` on SVG elements.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct
